src/modules/article/+state/article.selectors.ts

```diff
@@ -0,0 +1,22 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+import {
+  articleFeatureKey,
+  ArticleState,
+} from 'src/modules/article/+state/article.reducer';
+
+const getArticleState = createFeatureSelector<ArticleState>(articleFeatureKey);
+
+const getArticle = createSelector(
+  getArticleState,
+  (state) => state[articleFeatureKey].currentArticle
```

Comment:
In this case we will have some like this:

```suggestion
  (state) => state.currentArticle
```